source/modules/cms_ev_battery_health/source/tests/infrastructure/__snapshots__/test_snapshot/test_cms_ev_battery_health_snapshot.json

```diff
@@ -166,7 +166,7 @@
             "Arn"
           ]
         },
-        "Runtime": "python3.9",
+        "Runtime": "python3.11",
```

Comment:
is this runtime supposed to be bumped?

Reply:
This is a CDK generated lambda. The runtime update comes from a cdk update.